test/lib/getFormSubmission.test.ts

```diff
@@ -0,0 +1,61 @@
+import { vi, describe, it, expect, beforeEach } from "vitest";
+import { mockClient } from "aws-sdk-client-mock";
+import { DynamoDBDocumentClient, GetCommand } from "@aws-sdk/lib-dynamodb";
+import { getFormSubmission } from "../../src/lib/vault/getFormSubmission";
```

Comment:
I have not yet been able to simplify imports using the Typescript defined paths ("@lib/", "@routes/", ..)